code/game/objects/structures/inflatable.dm

```diff
@@ -439,3 +439,41 @@
 			update_icon()
 			to_chat(user,"<span class='notice'>You climb free of the shelter.</span>")
 
+/obj/structure/inflatable/shelter/MouseDropTo(atom/movable/O as mob|obj, mob/user as mob) //copy pasted from cryo code
+	if(!ismob(O))
```

Comment:
if you're going to check to make sure O is a living down at 464 this is redundant